core/src/it/scala/org/ensime/intg/BasicWorkflow.scala

```diff
@@ -86,7 +86,23 @@ class BasicWorkflow extends EnsimeSpec
           //-----------------------------------------------------------------------------------------------
           // symbolAtPoint
           project ! SymbolAtPointReq(Left(fooFile), 128)
-          val symbolAtPointOpt: SymbolInfo = expectMsgType[SymbolInfo]
+          expectMsgType[SymbolInfo]
+
+          project ! SymbolAtPointReq(Left(fooFile), 68) // scala.Int symbol
+          val sourcePosition = expectMsgType[SymbolInfo].`type`.pos.get
+          val sourceFile = sourcePosition match {
+            case OffsetSourcePosition(file, _) =>
+              file
+            case _ =>
+              EnsimeFile("empty")
+          }
+
+          project ! TypecheckFilesReq(List(Right(SourceFileInfo(sourceFile, None, None))))
+          expectMsg(VoidResponse)
+          asyncHelper.expectMsg(FullTypeCheckCompleteEvent)
+
+          project ! SymbolAtPointReq(Right(SourceFileInfo(sourceFile, None, None)), 720)
```

Comment:
we shouldn't be considering the `empty` case... if that doesn't work something failed. I guess typecheck file should be enough. But maybe asking for the semantic information instead of the symbol-at-point would be better, and just assert we get an non-empty response.